static/scss/_button.scss

```diff
@@ -22,7 +22,7 @@
   &.minor-action {
     font-size: 13px;
     padding: 9px 19px;
-    border: 1px solid #e6e6e6;
+    border: 1px solid #d0d0d0;
```

Comment:
I changed this to $border-color, and added this variable to the minor-button and the pagination and sort controls.